device-types/Supermicro/SYS-2028TP-HC1R-SIOM.yml

```diff
@@ -0,0 +1,26 @@
+---
+manufacturer: Supermicro
+model: SYS-2028TP-HC1R-SIOM
+slug: supermicro-sys-2028tp-hc1r-siom
+part_number: SYS-2028TP-HC1R-SIOM
+subdevice_role: child
+front_image: true
+rear_image: true
```

Comment:
1. Please add the `weight` and associated `weight_unit` data point if you have it or can find it. This appears to be listed in the datasheet (72 lbs or 32.7 kg)